src/config/2022.json

```diff
@@ -39,34 +56,21 @@
         {
           "part": "I",
           "chapter_number": "5",
-          "title": "Fonts",
-          "slug": "fonts"
-        },
-        {
-          "part": "I",
-          "chapter_number": "6",
```

Comment:
What's going on here? Looks like rest of this file messed up (copy and paste error?)